part4/blog/.env.example

```
@@ -1,3 +1,4 @@
 MONGODB_PW={yourPassword}
 MONGODB_URI=mongodb+srv://{userName}:{yourPassword}@{clusterUrl}/{dbName}?appName={clusterName}
-PORT=3001
\ No newline at end of file
+PORT=3001
+SECRET='your_secret_string'
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)